localisation/ME_Timurid_Princes_l_english.yml

```diff
@@ -88,17 +88,30 @@
  ME_Timurid_Princes_Event15.EVTOPT5: "A military advisor should do for now."
  ME_Timurid_Princes_Event15.EVTOPT6: "I've changed my mind."
  ME_Timurid_Princes_Event.EVTNAME16: "[TIM.Monarch.GetName] Demands Subjugation"
- ME_Timurid_Princes_Event.EVTDESC16: "The pretender to Tamerlane's Legacy has sent a scathingly disrespectful letter by messenger to the palac4e of $CAPITAL$. He demands our complete servitude. Despite the obvious lack of respect, we promises to ensure our territorial gain and mutual benefit. How should we respond?"
+ ME_Timurid_Princes_Event.EVTDESC16: "The pretender to Tamerlane's Legacy has sent a scathingly disrespectful letter by messenger to the palace of $CAPITAL$. He demands our complete servitude. Despite the obvious lack of respect, we promises to ensure our territorial gain and mutual benefit. How should we respond?"
  ME_Timurid_Princes_Event16.EVTOPT1: "We shall accept his demands."
  ME_Timurid_Princes_Event16.EVTOPT2: "Bring it on, pretender scum!"
  ME_Timurid_Princes_Event.EVTNAME19: "The Beijing-Samarkand Pact"
  ME_Timurid_Princes_Event.EVTDESC19: "The Chinese offer us a chance at partition. Should we accept, we would expect minor territorial gain as well as security from Chinese border raids. However, there lies a stipulation..."
  ME_Timurid_Princes_Event19.EVTOPT1: "We shall accept this offer."
  ME_Timurid_Princes_Event19.EVTOPT2: "Never!"
- ME_Timurid_Princes_Event.EVTNAME20: "Stampeding the Traitor"
- ME_Timurid_Princes_Event.EVTDESC20: "Baysonqor lies weakened by war and riddled with disease. Now may be the time to strike, lest our opportunity pass us by."
- ME_Timurid_Princes_Event20.EVTOPT1: "The traitor's reign shall end now!"
- ME_Timurid_Princes_Event20.EVTOPT2: "A bold move, I shall be more careful with this plan."
+ ME_Timurid_Princes_Event.EVTNAME20: "Pacifying the Traitor"
+ ME_Timurid_Princes_Event.EVTDESC20: "While we wait for our decisive victory [TIM.Monarch.GetName], a single question remains. How should we deal with this rebellious region forwards?"
+ ME_Timurid_Princes_Event20.EVTOPT1: "Bring them back into the fold under a loyal subject"
+ ME_Timurid_Princes_Event20.EVTOPT2: "A change of governance is in order!"
+ 
```

Comment:
These empty lines look out of place in this file now.

Reply:
For viewing purposes I like having them separated but I can remove it if its preferable

Reply:
It's definitely understandable to do when you are in the process of writing your code, but submitted should be as clean and consistent as possible. Please change.